q2_fondue/sequences.py

```diff
@@ -223,10 +223,9 @@ def _write_empty_casava(read_type, casava_out_path):
     Warns about `read_type` sequences that are not available
     and saves empty casava file.
     """
-    warn_msg = f'No {read_type}-read sequences available ' \
-               f'for these accession IDs.'
-    warn(warn_msg)
-    LOGGER.warning(warn_msg)
+    LOGGER.info(
+        'No %s-end sequences available for these accession IDs.', read_type
+    )
```

Comment:
See issue #91.

Reply:
In the most recent manuscript reviewer response we mentioned this is a warning. How about we keep it this way for now?

Reply:
I reverted to the previous version but removed the additional warning that was being raised above the logger.